src/main/resources/alma/fix/relatedRessourcesAndLinks.fix

```diff
@@ -152,13 +156,19 @@ do list(path: "830??", "var": "$i")
       copy_field("$i.w", "isPartOf[].$last.hasSuperordinate[].$last.id")
       replace_all("isPartOf[].$last.hasSuperordinate[].$last.id", "^\\(DE-605\\)(.*)$", "http://lobid.org/resources/$1#!")
     end
-    copy_field("$i.a", "isPartOf[].$last.hasSuperordinate[].$last.label")
+    set_array("isPartOf[].$last.hasSuperordinate[].$last.label", " / ")
+    do list(path:"$i.a", "var":"$j")
+      copy_field("$j", "isPartOf[].$last.hasSuperordinate[].$last.label.$append")
+    end
+    join_field("isPartOf[].$last.hasSuperordinate[].$last.label", " / ")
```

Comment:
The subfield $a only exists once. But the join_field behaves like there are multiple subfields $a.

Reply:
You're joining the initial value `" / "` and the subfield value `"Studia Judaica"` with your separator `" / "`, so the output `" /  / Studia Judaica"` is correct, isn't it?

To illustrate the individual parts: `"(?<initial value> / )(?<separator> / )(?<subfield value>Studia Judaica)"`